meilisearch-release-changelog/v1.13.0.md

```diff
@@ -0,0 +1,72 @@
+
+### Breaking Changes before Stabilization
+
+* Accepted Ollama URLs can only end with `/api/embed` and `/api/embeddings`.
+* Modified error codes:
+  - `invalid_embedder` has been split into `invalid_search_embedder` and `invalid_similar_embedder`. These codes are returned when the `embedder` parameter of a search (resp. similar) request refers to a non-existing embedder configuration or is not a string.
+  - `invalid_hybrid_query` has been renamed to `invalid_search_hybrid_query`. It is returned when the `hybrid` parameter is invalid: contains unknown keys or is not either `null` or an object.
+
+Done by @dureuill in https://github.com/meilisearch/meilisearch/pull/5232 & https://github.com/meilisearch/meilisearch/pull/5234
+
+## Dumpless Upgrade: Ease upgrading to the next version without a dump
+
+TBD by tamo
+
+## Proxy search requests
+
+TBD by Louis
+
+## Other improvements
+
+* Improve the auto-batching of tasks by exposing a way to limite the total size of a batch by @Kerollmops in https://github.com/meilisearch/meilisearch/pull/5223
+* Related to [Prometheus the experimental feature](https://github.com/orgs/meilisearch/discussions/625): Add Prometheus metrics to measure the task queue latency by @takaebato in https://github.com/meilisearch/meilisearch/pull/5178
+* Speed up the request to list the indexes by @irevoire in https://github.com/meilisearch/meilisearch/pull/5166
+* Add support for `GITHUB_TOKEN` authentication in installation script by @Sherlouk in https://github.com/meilisearch/meilisearch/pull/5216
+
+# Fixes 🐞
+
+* Improve the error message when an attributes is not filterable by @jameshiew in https://github.com/meilisearch/meilisearch/pull/5135
+
+# Misc
+
+* Dependencies updates
+  * Upgrade dependencies and fixes the `idna` severity issue by @Kerollmops in https://github.com/meilisearch/meilisearch/pull/5218
+* CIs and tests
+  * Split Meilisearch crate tests in separate file by @K-Kumar-01 in https://github.com/meilisearch/meilisearch/pull/5134
+  * Split tests in separate file by @K-Kumar-01 in https://github.com/meilisearch/meilisearch/pull/5171
+  * Remove obsolete test code by @K-Kumar-01 in https://github.com/meilisearch/meilisearch/pull/5173
+  * Fix the flaky batches test by @irevoire in https://github.com/meilisearch/meilisearch/pull/5175
+  * Split tests for option crate meilisearch in a separate test file by @K-Kumar-01 in https://github.com/meilisearch/meilisearch/pull/5174
+  * Remove hard coded task IDs to prevent flaky tests by @mhmoudr in https://github.com/meilisearch/meilisearch/pull/5182
```

Comment:
```suggestion
  * Remove hard coded task IDs to prevent flaky tests by @mhmoudr in https://github.com/meilisearch/meilisearch/pull/5182
  * Improve test performance of get_index.rs by @DerTimonius in https://github.com/meilisearch/meilisearch/pull/5210
```